pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-consent-assent-pdf-v3.conf

```diff
@@ -0,0 +1,132 @@
+{
+  "pdfs": [
+    {
+      "name": "osproject-consent-assent-v2",
+      "filename": "osproject-consent-assent-v2",
+      "displayName": "Osteo parental consent & assent pdf v2",
```

Comment:
display name can be updated to v3 to reflect activity version. 
Not mandatory though.